qiskit/providers/__init__.py

```diff
@@ -110,6 +109,394 @@
    Job
    JobV1
 
+======================
+Writing a New Provider
+======================
+
+If you have a quantum device or simulator that you would like to integrate with
+Qiskit you will need to write a provider. A provider will provide Terra with a
+method to get available :class:`~qiskit.provider.BackendV1` objects. The
+:class:`~qiskit.provider.BackendV1` object provides both information describing
+a backend and its operation for the :mod:`~qiskit.transpiler` so that circuits
+can be compiled to something that is optimized and can execute on the
+backend and also the :meth:`~qiskit.providers.BackendV1.run` method which can
```

Comment:
This sentence became hard to follow, especially since it says both but then continues onto a third thing
```suggestion
backend. It also provides the :meth:`~qiskit.providers.BackendV1.run` method which can
```